pkg/apis/batch/validation/validation.go

```diff
@@ -176,6 +176,14 @@ func ValidateCronJob(scheduledJob *batch.CronJob) field.ErrorList {
 	return allErrs
 }
 
+func ValidateCronJobUpdate(job, oldJob *batch.CronJob) field.ErrorList {
+	allErrs := apivalidation.ValidateObjectMetaUpdate(&job.ObjectMeta, &oldJob.ObjectMeta, field.NewPath("metadata"))
+	allErrs = append(allErrs, ValidateCronJobSpec(&job.Spec, field.NewPath("spec"))...)
+	// skip the 52-character name validation limit on update validation
```

Comment:
Do we need to sunset this skip in a future release?

Reply:
could, although the migrate strategy is pretty deployment-specific (ranging from "not at all" to "every release" to "tools available but not auto-run"), so it's unclear when we'd know they'd all been cleared out

Reply:
Hopefully the resolution to #52185 is "every release", but I agree with you re the status quo.

Reply:
hope is a prereq for all my upgrades

Reply:
@liggitt how long do we want to wait before "fixing" the update part entirely? Is 1.10 a good time frame or 1.11?